robottelo/ui/contentviews.py

```diff
@@ -55,18 +55,13 @@ def create(self, name, label=None, description=None, is_composite=False):
         self.wait_for_ajax()
         self.click(common_locators['create'])
 
-    def delete(self, name, really=True):
+    def delete(self, name):
```

Comment:
It is only one view that has this kind of form confirmation and `self.delete_entity()` handle only positive scenario. Negative scenario is not needed right now and if needed then we will think about proper solution